packages/v4/src/content/design-guidelines/usage-and-behavior/alerts-and-notifications/alerts-and-notifications.md

```diff
@@ -9,119 +9,124 @@ Alerts are used when the system wants to notify the user about some status or ot
 
 ## Elements
 ### Alert
-Alerts are used to communicate status information to the user without blocking other actions through the user interface. There are two styles that can be applied to an alert depending on whether it is intended as an [inline](#inline-alerts) or [toast](#toast-alerts) alert. Regardless of the styling, the basic elements of all alerts are consistent. Since toast alerts should always be dismissable, only a subset of variations should be used.
+Alerts are used to communicate status information to the user without blocking other actions through the user interface. There are two styles that can be applied to an alert depending on whether it is intended as an [inline](#inline-alerts) or [toast](#toast-alerts) alert. Regardless of the styling, the basic elements of all alerts are consistent.
 
-![alert types](./img/alert-types-2.png)
+<img src="./img/alert-types.png" alt="Alert types" width="868"/>
+
+Toast alerts should always be dismissable, whereas inline alerts can be both dismissable and non-dismissable
```

Comment:
I think this sentence could be easily missed and should be placed above the picture as a continuation of “Regardless of the styling, the basic elements of all alerts are consistent.”

So you'd end up with something like “Regardless of the styling, the basic elements of all alerts are consistent. The only difference is that toast alerts should always be dismissable, whereas inline alerts can be both dismissable and non-dismissable” 